test/controllers/CommonTaskListControllerSpec.scala

```diff
@@ -0,0 +1,137 @@
+
+import models.common.JourneyStatus
+import models.taskList.{SectionTitle, TaskListSection, TaskListSectionItem, TaskTitle}
+import org.scalamock.handlers.CallHandler5
+import play.api.http.Status.OK
+import play.api.libs.json.{JsValue, Json}
+import play.api.test.Helpers.{contentAsJson, status}
+import services.CommonTaskListService
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.mocks.MockAuthorisedAction
+import utils.providers.FakeRequestProvider
+import utils.{ControllerUnitTest, TaxYearUtils}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class CommonTaskListControllerSpec extends ControllerUnitTest with MockAuthorisedAction with FakeRequestProvider {
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
+  val nino: String = "123456789"
+  val taxYear: Int = TaxYearUtils.taxYear
+
+  val commonTaskListService: CommonTaskListService = mock[CommonTaskListService]
+  val controller = new CommonTaskListController(commonTaskListService, mockAuthorisedAction, cc)
+
+  def taskListSection(title: SectionTitle, section: Option[Seq[TaskListSectionItem]]): TaskListSection = TaskListSection(title, section)
+
+  val ukItem: Option[Seq[TaskListSectionItem]] = Some(Seq(TaskListSectionItem(TaskTitle.UkProperty, JourneyStatus.Completed, Some("TEST"))))
+  val foreignItem: Option[Seq[TaskListSectionItem]] = Some(Seq(TaskListSectionItem(TaskTitle.ForeignProperty, JourneyStatus.Completed, Some("TEST"))))
+  val ukForeignItem: Option[Seq[TaskListSectionItem]] = Some(Seq(TaskListSectionItem(TaskTitle.UkForeignProperty, JourneyStatus.Completed, Some("TEST"))))
+
+  val returnJsonNone: JsValue = Json.parse(
+    """
+      |[
+      |  {
+      |    "sectionTitle": "UkProperty"
+      |  },
+      |  {
+      |    "sectionTitle": "ForeignProperty"
+      |  },
+      |  {
+      |    "sectionTitle": "UkForeignProperty"
+      |  }
+      |]
+      |""".stripMargin
+  )
+
+  val returnJsonSome: JsValue = Json.parse(
+    """
+      |[
+      |  {
+      |    "sectionTitle": "UkProperty",
+      |    "taskItems": [
+      |      {
+      |        "title": "UkPropertyTitle",
+      |        "status": "completed",
+      |        "href": "TEST"
+      |      }
+      |    ]
+      |  },
+      |  {
+      |    "sectionTitle": "ForeignProperty",
+      |    "taskItems": [
+      |      {
+      |        "title": "ForeignPropertyTitle",
+      |        "status": "completed",
+      |        "href": "TEST"
+      |      }
+      |    ]
+      |  },
+      |  {
+      |    "sectionTitle": "UkForeignProperty",
+      |    "taskItems": [
+      |      {
+      |        "title": "UkForeignPropertyTitle",
+      |        "status": "completed",
+      |        "href": "TEST"
+      |      }
+      |    ]
+      |  }
+      |]
+      |""".stripMargin
+  )
+
+  def mockStateBenefitsService(taskListSection: Seq[TaskListSection]): CallHandler5[Int, String, String, ExecutionContext, HeaderCarrier, Future[Seq[TaskListSection]]] = {
```

Comment:
Think this should be `property` instead of `state benefits`:
```suggestion
  def mockPropertyService(taskListSection: Seq[TaskListSection]): CallHandler5[Int, String, String, ExecutionContext, HeaderCarrier, Future[Seq[TaskListSection]]] = {
```